app.js

```diff
@@ -1,16 +1,55 @@
-/* Imports */
+const computerFight = ['Jackson', 'Spongebob', 'Patrick'];
+
+function getRandomNumber(choices) {
+    return Math.floor(Math.random() * choices);
+}
+
+function generateFight(array) {
+    const random = getRandomNumber(array.length);
+    const item = array[random];
+    return item;
+}
 
+/* Imports */
+const SpongebobFight = document.getElementById('Spongebob');
+const PatrickFight = document.getElementById('Patrick');
+const JacksonFight = document.getElementById('Jackson');
+const results = document.getElementById('results');
+const playerFightImg = document.getElementById('playerFight');
+const computerFightImg = document.getElementById('computerFight');
 /* State */
 
 /* Actions */
 function loadPage() {}
+function runGame(player) {
+    const computerResult = generateFight(computerFight);
+
+    console.log(computerResult);
+    console.log(player);
+
+    results.classList.remove('hidden');
+
+    playerFightImg.src = `${player}.jpg`;
+    computerFightImg.src = `${computerResult}.jpg`;
+    console.log(playerFightImg);
+}
 
 /* Components */
 
 /* Component */
+
 // get DOM
+
 // display
-// event listeners
 
-/* Run page load code */
+// event listeners
+SpongebobFight.addEventListener('click', () => {
+    runGame('Spongebob');
+});
```

Comment:
this is where you'd want to be using state — when the user clicks, we update state. so you could have a piece of state that was like ```let userGuess = ''``` and then when they click on rock you say ```userGuess = 'rock'``` and then you could call ```runGame(userGuess)```